dev-haskell/zlib/Manifest

```diff
@@ -1,2 +1,4 @@
 DIST zlib-0.6.3.0-rev5.cabal 5465 BLAKE2B 79a7f113e8cc9361c4f4c85edfb5017009ee7d14a1b28e62bf11b88614e88e8938d45eeccb1dc9c2b4d5dae1d1f1da68a6c9d7b5dd292d3f4cbc6d3e2b807699 SHA512 d7abbed0ed07991c68f6c9b6f6afac1b1962cc9b4c046b286af00662cfad291c36e9ded1128a8ff034f23edd8fb405a6e83f533460a7bd41ae234e7b07f0d8ab
 DIST zlib-0.6.3.0.tar.gz 369595 BLAKE2B 3a3ecb4ef2cce3332be22a1137b89c3ff27a600326fbc4c53610a588bcfe79f2c3b6916ad68705b7d397c0cc53aaf958a19eccd39edfb66e07b3cfae701d8890 SHA512 95daae13c5d5458fd7c610ce4cc686dc7af477364c4dcf3d31abede84f6e4b4e70ea255f7ded586d46d9bb6298d19c93b8856eec16c6002265ed076e52bf9a21
+DIST zlib-0.7.1.0-rev2.cabal 5357 BLAKE2B bf84ade33e213a2ad7a6bb144c3cb68e98587b1ae2e8eea37561d93c59f2a25b5f0ec3894546a3a0c85a1d3d114f3d62d8c81ff4d3dcc7468f1770c39edba7c2 SHA512 ced6d37d7155fcccef9512c5a23f3f40b58dcf7e2b9c6e4d7df4828e6b1fcaf88242b4dd0e6db07f3b5bc4cce051cdeca1f925660e6c9dc43efac0e2f3caca51
```

Comment:
Just curious, why are you leaving `zlib-0.6.3.0`?

Reply:
I assume this is because it's a major bump and you're being cautious (and saving yourself extra work.) Not a problem!